src/service/ui/assets/js/video.js

```diff
@@ -1,30 +1,30 @@
 ;(function() {
+	var videoId
+	var duration
+
 	var notify = function(elapsed) {
-		var pct = elapsed / VIDEO_DURATION * 100
+		var pct = elapsed / duration * 100
 		if (pct > 100) {
 			pct = 100
 		} else {
 			pct = Math.floor(pct)
 		}
 		console.log('Progress: ' + pct + '%')
 	}
+
+	videoId = document.getElementById('video-id').value
+
 	window.onYouTubeIframeAPIReady = function() {
 		var player = new YT.Player('video-player', {
-			height: VIDEO_HEIGHT,
-			videoId: VIDEO_ID,
-			width: VIDEO_WIDTH,
+			height: 390,
+			videoId: videoId,
+			width: 640,
```

Comment:
Note these are the defaults, but left here to indicate how we can adjust size to match any UI designs. Don't want to adjust video based on the content as we need it to fit our screens.